### DISCUSSION TITLE
I Cannot visit the versel.com website

### DISCUSSION THREAD

**Discussion by emon3455** [vercel/community#3850] [Account Issue]
### Question

I cannot visit vercel.com website from my network but by using vpn i can access the website. In this situation what can i do. If anyone know please help me out.

**Comment by swarnava** [comment]
Hi, Are you working on University network or something? They likely blocking the traffic. You can check with local ISP or university's IT team for more info.

**Reply by emon3455** [reply]
no i am working in my home not in university

**Reply by swarnava** [reply]
Can you share a screenshot?
